ci-operator/config/openshift-priv/installer/openshift-priv-installer-main.yaml

```diff
@@ -1043,9 +1043,13 @@ tests:
   steps:
     cluster_profile: vsphere-elastic
     env:
+      ADDITIONAL_ENABLED_CAPABILITIES: MachineAPI CloudControllerManager Ingress CloudCredential
```

Comment:
@sgaoshang @WenXinWei any concerns with the strategy of disabling some capabilities for vSphere presubmits?

Reply:
as [doc](https://docs.redhat.com/en/documentation/openshift_container_platform/4.19/html/installation_overview/cluster-capabilities) mentioned, admin can disable optional components prior to installation, and enable after installation. so i thinks it looks good to me.